src/repl/QuackRepl.php

```diff
@@ -20,7 +20,7 @@
  * along with Quack.  If not, see <http://www.gnu.org/licenses/>.
  */
 define('BASE_PATH', __DIR__ . '/..');
-require_once '../toolkit/QuackToolkit.php';
+require_once(BASE_PATH.'/toolkit/QuackToolkit.php');
```

Comment:
Well, the only issue is that our pattern for operators is:
 `<expr> <op> <expr>`, which is space separated: `BASE_PATH . '/toolkit...`.